gradle-stage0.yaml

```diff
@@ -1,13 +1,13 @@
 package:
   name: gradle-stage0
   version: 8.0.1
-  epoch: 0
+  epoch: 1
   description: A Java project management and project comprehension tool.
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
-      - openjdk-11-jre
+      - openjdk-11-default-jvm
```

Comment:
meta: we need to decide whether we runtime include jvm or not.  @kaniini what happens if two packages in the dependency tree provide default-jvm but for different jdk versions?